shared/src/httpClient/errors/httpErrors.schema.ts

```diff
@@ -0,0 +1,7 @@
+import { z } from "zod";
+
+export const errorSchema = z.object({
```

Comment:
on peut renommer peut-être, je te propose :

```suggestion
export const httpErrorSchema = z.object({
```